code/modules/surgery/robotics.dm

```diff
@@ -360,6 +366,7 @@
 /singleton/surgery_step/robotics/attach_organ_robotic/begin_step(mob/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
 	user.visible_message("<b>[user]</b> begins reattaching [target]'s [target.op_stage.current_organ] with \the [tool].", \
 		SPAN_NOTICE("You start reattaching [target]'s [target.op_stage.current_organ] with \the [tool]."))
+	playsound(target.loc, tool.surgerysound, 15, TRUE)
 	..()
```

Comment:
Most of these call ..(), just shove the playsound in the parent `begin_step()` and make the ones that don't call ..() actually call it

Reply:
Will do!